pkg/virt-controller/watch/vm.go

```diff
@@ -2904,6 +2990,13 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 			}
 		}
 	}
+
+	_, err = c.createVMRevision(vm, revisionPrefixLastSeen)
```

Comment:
I was thinking that creating one on start/restart is enough for our use-case, WDYT?

Reply:
That's a really good point. In the current state of things you're right, the boot-time VM spec can be used as a reference to decide if RestartRequired is needed.
However, the follow up PR to this will need the last seen spec to save it as the last non-RestartRequired generation to add the ability to get rid of RestartRequired without restarting (i.e. correct a mistake for example).
In the meantime you're right, we shouldn't add unnecessary overhead, I'll work on a fix.

Reply:
Fixed